site/src/components/Playground.tsx

```diff
@@ -0,0 +1,190 @@
+import * as React from 'react'
+import { gray } from '@radix-ui/colors'
+
+export function Playground({
+  ts,
+  kyselyVersion = '0.23.3',
```

Comment:
imo we'd need to find a solution to make this future proof,
maintaining a version number here is risky, it's easy to forget

Reply:
This should be auto-generated in release workflow maybe?